spec/audited/sweeper_spec.rb

```diff
@@ -1,6 +1,12 @@
 require "spec_helper"
 
 class AuditsController < ActionController::Base
+  if Rails::VERSION::MAJOR == 4
+    before_filter :populate_user
```

Comment:
Ah, you're right - thanks for the info. I've updated the PR to remove the conditional entirely.